packages/firebase_mlkit_language/lib/src/firebase_language.dart

```diff
@@ -0,0 +1,34 @@
+part of firebase_mllanguage;
+
+class FirebaseLanguage {
```

Comment:
I think we should call this FirebaseNaturalLanguage to match naming of the native SDKs. It seems that the word "natural" is used consistently everywhere.